e2e/suites/application/page-titles.test.ts

```diff
@@ -25,13 +25,23 @@
 
 import { browser } from 'protractor';
 
-import { SIDEBAR_LABELS } from '../../configs';
+import { SIDEBAR_LABELS, PAGE_TITLES } from '../../configs';
 import { LoginPage, LogoutPage, BrowsingPage } from '../../pages/pages';
+import { RepoClient } from '../../utilities/repo-client/repo-client';
+import { Utils } from '../../utilities/utils';
+import { protractor } from '../../../node_modules/protractor/built/ptor';
 
 describe('Page titles', () => {
     const loginPage = new LoginPage();
     const logoutPage = new LogoutPage();
     const page = new BrowsingPage();
+    const apis = {
```

Comment:
you only have a single api here
const adminApi = new RepoClient() would be enough